test_gui.py

```
@@ -4,13 +4,14 @@
 import json
 from pathlib import Path
 import shutil
+import streamlit as st
```

Ruff (0.8.2)

`streamlit` imported but unused

Remove unused import: `streamlit`

(F401)